app/concerns/owable.rb

```diff
@@ -3,19 +3,19 @@ module Owable
 
   included do
     has_many :post_authors, inverse_of: :post, dependent: :destroy
-    has_many :authors, class_name: 'User', through: :post_authors, source: :user
+    has_many :authors, class_name: 'User', through: :post_authors, source: :user, dependent: :destroy
 
     # list of users who owe replies on this post, whether or not they have posted yet
     has_many :tagging_post_authors, -> { where(can_owe: true) }, class_name: 'PostAuthor', inverse_of: :post
```

Comment:
Does this also want `dependent: :destroy`, along with the other similar `_post_` things below?

Reply:
There are other places that do this, too, and I get the feeling we want more `dependent: :destroy`s there (board coauthors)?

Reply:
I think it does not matter here because it's scoped on the `has_many :post_authors`, which does; since this is one way (deleting a PostAuthor does not delete a Post, but having the unscoped declaration means deleting a Post will always delete all PostAuthors, therefore including scoped PostAuthors) we should be fine?

Reply:
Makes sense! It would be super weird to do `.tagging_post_authors = [list, of, post_authors]` anyway, and that's where this comes into play.